pallets/creditcoin/src/tests/collectCoins.json

```diff
@@ -473,5 +473,10 @@
       "transactionsRoot": "0x4270c05aff0c30786ff65957ed7b2d6c964482b5acc9bfc18e12f1aa4736b453",
       "uncles": []
     }
+  },
+  "eth_chainId": {
```

Comment:
I don't think we use `eth_chainId` in collect_coins

Reply:
Yeah, it's just that right now in `MockedRpcRequests` we unconditionally populate all of the fields so it was a choice between reworking `MockedRpcRequest` a fair bit or just putting `eth_chainId` in here, even though it's unused